packages/shared/src/components/image/VideoImage.tsx

```diff
@@ -10,7 +10,7 @@ export interface VideoImageProps extends ImageProps {
 }
 
 const VideoImage = ({
-  size = IconSize.XXXLarge,
+  size = IconSize.XLarge,
```

Comment:
Does this mix up the size for the video images on cards? I think they require the XXXLarge by default?